driver-core/src/main/com/mongodb/AutoEncryptionSettings.java

```diff
@@ -185,6 +193,42 @@ public Builder bypassAutoEncryption(final boolean bypassAutoEncryption) {
             return this;
         }
 
+        /**
+         * Maps a collection namespace to an encryptedFields.
+         *
+         * <p><strong>Note:</strong> only applies to FLE 2. Automatic encryption in FLE 2 is configured with the encryptedFields.
+         * <p>If a collection is present in both the {@code encryptedFieldsMap} and {@link #schemaMap}, libmongocrypt will error on
+         * initialization.
+         * <p>If a collection is present on the {@code encryptedFieldsMap}, the behavior of {@code createCollection()} and
+         * {@code collection.drop()} is altered.
+         *
+         * <p>If a collection is not present on the {@code encryptedFieldsMap} a server-side collection {@code encryptedFieldsMap} may be
+         * used by libmongocrypt.
+         *
+         * @param encryptedFieldsMap the mapping of the collection namespace to the encryptedFields
+         * @return this
+         * @since 4.7
+         */
+        public Builder encryptedFieldsMap(final Map<String, BsonDocument> encryptedFieldsMap) {
+            this.encryptedFieldsMap = notNull("encryptedFieldsMap", encryptedFieldsMap);
+            return this;
+        }
+
+        /**
```

Comment:
None of the other builder setters define a default - so not sure why this should be different. 

Also this is complicated because as saying the default here (an empty map)  doesn't mean no fields are encrypted: 

> If a collection is not present on the encryptedFieldsMap a server-side collection encryptedFieldsMap may be used by the driver.

Reply:
bypassAutoEncryption doesnt define the default - I think the reasoning is the name bypass is implicitly false.